### ISSUE TITLE
Headland Management Support?

### ISSUE THREAD

**Issue by cwhandersen** [TobiasF92/FS22_interactiveControl#15]
Hello
Awesome mod. I absolutely love it.
Would you be able to add support for Headland Management?
So a toggle option like Guidance Steering.

And could you tell me how to convert a mod from Simple IC to Interactive Control?

**Comment by TobiasF92** [comment]
Thank you :) 

Can you provide me the mod/link of "Headland Management"?

**Comment by cwhandersen** [comment]
Here's the link for Headland Management:
https://www.farming-simulator.com/mod.php?lang=en&country=us&mod_id=228759&title=fs2022

**Comment by TobiasF92** [comment]
ty

> It can be activated and deactivated in the vehicle with Shift-X.
thats the thing you want to control via IC?

**Comment by cwhandersen** [comment]
Yes. 
I think Fendt has a system like it IRL. So I would love to have the ability to use it in game.

Also, would you mind if I uploaded a vehicle to modhub using your Interactive Control mod?

**Comment by TobiasF92** [comment]
> Also, would you mind if I uploaded a vehicle to modhub using your Interactive Control mod?

No, absoultely not. Just point out, that your mod requires IC for full functionality ;)
`- added 'interactive control' support (Requires 'FS22_interactiveControl')`

E.g.:
https://www.farming-simulator.com/mod.php?mod_id=225936

**Comment by TobiasF92** [comment]
Added in  [Beta3](https://github.com/TobiasF92/FS22_interactiveControl/releases/tag/1.1.0.0-beta3)
